From d222104a1a69a3ce1270e5b906b5074081826c32 Mon Sep 17 00:00:00 2001
From: Sebastien PASDELOUP <sebastien.pasdeloup-ext@st.com>
Date: Wed, 3 Jul 2024 11:49:16 +0200
Subject: [PATCH 642/841] feat(stm32mp21-fdts): add board ID OTP for
 STM32MP215F-DK

Add the board_id node in STM32MP215F-DK device tree file. This is used
on STMicroelectronics boards to have information on the board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 0194be5cbedbb6ac9bb6147d62c4d04a7ead57ad feat(stm32mp21-fdts): add board ID OTP for STM32MP215F-DK]
Change-Id: Iefb5db6334ea1146e1e7b11e5f98465090b5d1b9
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/390341
Domain-Review: Sebastien PASDELOUP <sebastien.pasdeloup-ext@st.com>
Tested-by: Sebastien PASDELOUP <sebastien.pasdeloup-ext@st.com>
Reviewed-by: Sebastien PASDELOUP <sebastien.pasdeloup-ext@st.com>
---
 fdts/stm32mp215f-dk.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/fdts/stm32mp215f-dk.dts b/fdts/stm32mp215f-dk.dts
index 97e8807f8..d0abd4d96 100644
--- a/fdts/stm32mp215f-dk.dts
+++ b/fdts/stm32mp215f-dk.dts
@@ -42,6 +42,12 @@
 	};
 };
 
+&bsec {
+	board_id: board_id@3d8 {
+		reg = <0x3d8 0x4>;
+	};
+};
+
 &ddr {
 	vdd1-supply = <&vdd1_ddr>;
 	vdd2-supply = <&vdd2_ddr>;
-- 
2.47.2

